torch/_functorch/autograd_function.py

```diff
@@ -145,11 +145,14 @@ def jvp(ctx, *tangents):
     )
     return Generated
 
+NO_OUT_DIMS = "not specified"
```

Comment:
This is to avoid confusing None with "not specified" right? Maybe we could have a comment about what happens in the None case?